source/includes/examples/cli-example-audit-logs-known-users.rst

```diff
@@ -0,0 +1,4 @@
+.. code-block::
+   :copyable: true
+
+   atlas auditing update --auditFilter '{"atype": "authenticate"}'
```

Comment:
Does this (and the TF auditing example) audit one of the items we recommended they audit? I don't see it in the list. I also expect it would have to specify the fields to return (the second list in the recommendations), is that not needed in the audit filter?

Reply:
@lantoli this is the comment I mean re: updating the example.